### DISCUSSION TITLE
Meta_Hours_5 - die Bohrmaschine

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#2210] [Meta-Hours]
Wir brauchen Meme für fünf Runden

Discord -- https://discord.com/events/840514076538830888/939090547333791754
Add to your calendar 

![image](https://user-images.githubusercontent.com/5887929/152516497-6df296f8-4414-49b3-bbeb-eba9f401dcff.png)

<img width="394" alt="Screenshot 2022-02-04 at 10 32 01" src="https://user-images.githubusercontent.com/5887929/152505796-42dac7f2-7e48-4a2d-af37-0319e144ee70.png">

### Time for wellbeing and paid recreation

<img width="400" src="https://user-images.githubusercontent.com/5887929/153565672-b712da28-0972-457c-8750-4f5b15e61501.png">

- issue to visualize Kodadot repository activity #1661
- you can check the previous visualization in [meta_hours_2](https://github.com/kodadot/nft-gallery/discussions/1699) 

### Short version
Yeah, **we believe it's not possible to keep up the current pace or rampage as we'd love to and every human being needs to rest a bit.**

Thus the upcoming days/weeks volume of updates will be less I guess and it's time to relax for us.

We are still welcome new contributors, artists and developers, yet we will be, at least I'll be in lazy mode :)

The best way to refuel your wellbeing is to take some time to unwind time before sleep, for best results to get quality sleep, turn down 🪫 electronics 3hrs prior to sleep to let your mind calm down.
My recommendation is to read the book, I recently got a kindle or play something, we've got Nintendo switch so if you're up doing multiplayer, we can have some gaming sessions together and casual chill chat, promise no crypto talk.

### Deeper version
I sense that the recent migration to Nuxt we've had and post-fixes **have been quite heavy on any recent contributor**, namely @roiLeo @vikiival @kkukelka @prachi00, me including @yangwao. 

Lot's of cosmetic issues, merge conflicts, errors, problems we had might easily deplete your energy levels under high stress. If I account you probably have all primary jobs or other side hustle running, degree of magnitude must be even harder and stronger. 

With ever coming issues we have and stuff to be fixed, it might feel like a lot's stuff to be fixed and is like an infinite rabbit hole. Some might enjoy it well, but some **challenging puzzles might easily squeeze you out of your best mood** while looking cracking on the issue. We all know software develops and quality is brewed by time and iterations we go through. Especially we are entering a super competitive crypto market, where major market turnarounds happen over the weekend, one needs to be steady to reflect and adapt quite quickly. But, **you are human and not machine**. 

Hence I can feel the pain as a founder and I've recently recovered from shallow burnout, don't want to repeat you go the same bottom. Despite all the stuff, I've come to ideation that **would be a superior time to get off after 3 months of intense grind.** 

### Incentivized retreat 
To incentivize this well-being retreat, it should be paid or give any sort of value for being. 
Thus we are all around the world not in one place (maybe one day if Subhouse would be a thing) **I really would like to see you more recreating and relaxing.** This doesn't mean we will not accept your pull request. I guess we will have just **more lazy mode**. 
Hence I see those upcoming months **might be double or triple on load**, would be more than welcome to be prepared for the next series of grinding stuff. 

To reflect on what have you accomplished is simply spectacular and breathtaking. Rephrasing it, **we've doubled monthly pull-requests** from the previous month. 

To be exact, **hitting 250 pull requests on monthly basis** is simply **athletic endurance**. Same as athletes, great accomplishment deserves great rest. To be prepared for the next win. We've even been noticed by media ranking KodaDot repository quite high. But you can't win other medals without rest. 

Thus on continue, **I will try to keep hanging around in Discord**, might we have an all-hands-devs call as we had earlier and chat a little bit about stuff we want to reflect on, share vision between each other, share your best learnings, what we can do better pimp for lowest effort bring most impactful pull-requests and refresh your ideas about NFTs, share your recent recreations and wellbeing upgrades and so on.

I can go first, that we have planned founders retreat in Norway (27/2-13/3), going for snowkiting, in middle of nowhere. With the closest grocery 30km close, I sense it would be an amazing opportunity for my/ours mental downtime. I'm really looking forward to it coming and refreshing my thoughts with a lot of reading and probably not interacting at all with the outside world.

![image](https://user-images.githubusercontent.com/5887929/154315004-71807264-25cc-49dd-937e-961a306211a6.png)

We will be for two weeks middle of nowhere, under the snow, hopefully, Internet would be at a place, if not, we don't worry, we should relax.

This relax window would help us to zoom out and check back on stuff we've might be missing for a pretty while long! :)

We will try to evaluate the stuff we want to do this year. Which might have an impact and so on.

**Thus measurements I was thinking to make this happen**
- Contributions are still welcome, yet PRs would be reviewed/tested approx once per day. Maybe less occurrence.
- There will be lower bound payouts as hence we value your contribution, it's come the **time to slow down the pace**, as higher bound of bounty was meant for example for fast delivery.
- Too frequent contributors named early, I was thinking payout significant amount to pay you time off as we value you as wellbeing and you can let me know voluntary how did you spend it, I'm really curious on this page.

Probably I can not think of any further thoughts on this, but this is most of it. 
**I value you as wellbeing, not a machine :)**
**If you can't enjoy contributing anymore, it slips to mercenary**. And I don't want many mercenaries much around, rather prefer missionaries who are aligned with the KodaDOt project vision.
Thus money reward for your contribution should be a side-effect, not the primary objective.

### Upcoming
- Updating documentation for easier onboarding new developers #2185 - https://github.com/kodadot/nft-gallery/pull/2212
- new release https://github.com/kodadot/nft-gallery/pull/2377

**Comment by damskyftw** [comment]
damsky's entry - will try to keep it short for today's edition

- got some updates regarding discord customization
- second part of customization incoming
- already got few artists on spotlight, more are welcome - with pushed curated feed by @roiLeo  it's going to be quite interesting
- docs are getting bigger and bigger - added [overview ](https://docs.kodadot.xyz/tutorial-overview.html#technology)for tutorials where we'll have up-to-date tuts
- kodadot x novawallet updates - made a [tutorial ](https://docs.kodadot.xyz/tutorials/how-to-kodadot-phone-ios.html#what-we-need)for those brave ones - gotta also make some understandable video about this
- also working on full beginner page for those interested in the [technology ](https://docs.kodadot.xyz/beginner-intro.html#is-this-page-for-you)- everything at one place - docs
- [first ](https://youtu.be/_b0xGiqwDcA)tutorial is up on KodaDot youtube! 
- [Subscribe ](https://www.youtube.com/channel/UCEULduld5NrqOL49k1KVjoA)for more videos :)

**Comment by vikiival** [comment]
**Viki's Meta Minutes**
-

### Subsquid:

With the help from the guys from Subsquid, we migrated our indexer to version 5, which brings a couple of new features:

We cache and process the data for spotlight and series insight. With the upcoming release, the queries should be almost instant. These data are updated every 10 minutes which is enough to reflect all buys and other changes.

Another important update is the collection chart resolver. As you may have seen, the chart on a big collection takes a lot of time to process. This is because we pass too much data to the chart engine. Our new approach aggregates data into the dates and you will see the nice, less noisy chart.

The third update is that we implemented md5 hashes to the NFTs. This is important because we want you to have profile pictures and banners on your profile.

### Dev docs:

We love to see the wild implementations of KodaDot and therefore we make easy setup for both our SubQuery indexers. For RMRK we use [magick](https://github.com/vikiival/magick) and for the Statemine NFTs we have [unique](https://github.com/kodadot/unique)

Documentation for the SubSquid indexer will come soon.
Also, we updated the documentation for the [first-time contributors](https://github.com/kodadot/nft-gallery/blob/main/FIRST_TIME.md) with a lot of examples that can help you to implement specific tasks in KodaDot.
